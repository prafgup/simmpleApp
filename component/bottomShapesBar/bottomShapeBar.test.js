import React from "react";
import {shallow} from "enzyme";

import BottomFancyBar from "./index";


describe("BottomFancyBar",()=>{

    describe("BottomFancyBar",()=>{
        let component ;

        beforeEach(()=>{
            component = shallow(<BottomFancyBar/>);
        });

        it("Should match to snapshot",()=>{
            expect(component).toMatchSnapshot("Name Bio");
        });

        it("Should Render all 4 Items",()=>{
            expect(component.find("[data-test='bottomBar']").length).toBe(4);
        });
    })
});