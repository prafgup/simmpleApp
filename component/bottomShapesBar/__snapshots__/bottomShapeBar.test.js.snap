// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomFancyBar BottomFancyBar Should match to snapshot: Name Bio 1`] = `
BottomFancyBar {
  "context": Object {},
  "props": Object {},
  "refs": Object {},
  "setState": [Function],
  "state": null,
  "updater": Updater {
    "_callbacks": Array [],
    "_renderer": ReactShallowRenderer {
      "_context": Object {},
      "_didScheduleRenderPhaseUpdate": false,
      "_dispatcher": Object {
        "readContext": [Function],
        "useCallback": [Function],
        "useContext": [Function],
        "useDebugValue": [Function],
        "useDeferredValue": [Function],
        "useEffect": [Function],
        "useImperativeHandle": [Function],
        "useLayoutEffect": [Function],
        "useMemo": [Function],
        "useReducer": [Function],
        "useRef": [Function],
        "useResponder": [Function],
        "useState": [Function],
        "useTransition": [Function],
      },
      "_element": <BottomFancyBar />,
      "_firstWorkInProgressHook": null,
      "_forcedUpdate": false,
      "_instance": [Circular],
      "_isReRender": false,
      "_newState": null,
      "_numberOfReRenders": 0,
      "_renderPhaseUpdates": null,
      "_rendered": <View
        style={
          Object {
            "alignContent": "center",
            "justifyContent": "center",
            "position": "relative",
          }
        }
      >
        <View
          data-test="bottomBar"
          style={
            Object {
              "borderColor": "orange",
              "borderWidth": 1,
              "height": 0,
              "position": "absolute",
              "top": 25,
              "width": "100%",
            }
          }
        />
        <View
          data-test="bottomBar"
          style={
            Object {
              "alignContent": "center",
              "backgroundColor": "white",
              "borderColor": "orange",
              "borderRadius": 22.5,
              "borderWidth": 2,
              "height": 45,
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": 45,
            }
          }
        />
        <View
          data-test="bottomBar"
          style={
            Object {
              "backgroundColor": "orange",
              "height": 45,
              "marginLeft": "10%",
              "position": "relative",
              "top": -5,
              "width": 45,
            }
          }
        />
        <View
          data-test="bottomBar"
          style={
            Object {
              "backgroundColor": "transparent",
              "borderBottomColor": "orange",
              "borderBottomWidth": 45,
              "borderLeftColor": "transparent",
              "borderLeftWidth": 22.5,
              "borderRightColor": "transparent",
              "borderRightWidth": 22.5,
              "borderStyle": "solid",
              "height": 0,
              "position": "absolute",
              "right": "10%",
              "top": 40,
              "width": 0,
            }
          }
        />
      </View>,
      "_rendering": false,
      "_updater": [Circular],
      "_workInProgressHook": null,
    },
  },
}
`;
